<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>assesment</class>
 <widget class="QWidget" name="assesment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>63</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="firstStar">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource/rsc.qrc">:/star_no_fill.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="secondStar">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource/rsc.qrc">:/star_no_fill.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="thirdStar">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource/rsc.qrc">:/star_no_fill.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fourthStar">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource/rsc.qrc">:/star_no_fill.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fifthStar">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource/rsc.qrc">:/star_no_fill.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource/rsc.qrc"/>
 </resources>
 <connections/>
</ui>
